packages/mdctl-axon-tools/packageScripts/ingestTransform.js

```diff
@@ -1,4 +1,10 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Can you add some tests for this? `TOOLS-44.test.js` has an example on how to test the ingest